src/devices/src/virtio/net/device.rs

```diff
@@ -56,6 +56,21 @@ fn init_vnet_hdr(buf: &mut [u8]) {
     }
 }
 
+#[derive(Clone, Copy)]
+pub struct ConfigSpace {
```

Comment:
Does it make sense to directly `Versionize` this instead of having a separate identical `NetConfigSpace` struct in net/persist.rs?

If we do keep them separate, it would be nice to rename `NetConfigSpace` to `NetConfigSpaceState`.

Reply:
I think we could follow the separate structures guidelines so we get consistency for all devices.

Reply:
Renamed it to `NetConfigSpaceState`.